Translated/Dialog/Souya/ssn_015310.csv

```diff
@@ -0,0 +1,11 @@
+name_7446#0,"""Souya"""
+ssn_015310_0001#0,"""In a protracted war scenario, supply replenishment, as well as management of said, is more important than anything else."""
```

Comment:
May read better saying

> supplying replenishments, as well as management of so

If you rather keep the 'said' wording, then perhaps

> as well as management of said items